scipy/special/_add_newdocs.py

```diff
@@ -1847,19 +1847,30 @@ def add_newdoc(name, doc):
     """)
 
 add_newdoc("chndtr",
-    """
+    r"""
     chndtr(x, df, nc, out=None)
 
     Non-central chi square cumulative distribution function
 
+    The cumulative distribution function is given by:
+
+    .. math::
+
+        P(\chi^{\prime 2} \vert \nu, \lambda) =\sum_{j=0}^{\infty}
+        e^{-\lambda /2}
+        \frac{(\lambda /2)^j}{j!} P(\chi^{\prime 2} \vert \nu + 2j),
+
+    where :math:`\nu > 0` is the degrees of freedom and :math:`\lambda \geq 0`
+    is the non-centrality parameter.
+
     Parameters
     ----------
     x : array_like
-        Upper bound of the integral
+        Upper bound of the integral; must have ``x >= 0``
```

Comment:
Not sure as I'm not a native english speaker but it seems to me that it would be better to say something like
```suggestion
        Upper bound of the integral; must satisfy ``x >= 0``
```

(true for all parameters below as well)